pywren/wrenhandler.py

```diff
@@ -59,6 +59,7 @@
 logger = logging.getLogger(__name__)
 
 PROCESS_STDOUT_SLEEP_SECS = 2
+CANCEL_CHECK_EVERY_SECS = 5.0
```

Comment:
Does this have to be hard-coded to 5, can we make this configurable?

Reply:
I think we can leave it hardcoded at 5 for now. It shouldn't be difficult to change and I don't see much benefit of allowing the user to specify it.